client_functions/orders.py

```diff
@@ -0,0 +1,23 @@
+async def create_funding(amount,per,rate,coin,bfx):
+  response = None
+  try:
+      response = await bfx.rest.submit_funding_offer('f{}'.format(coin), amount, rate, per)
+  except Exception as e:
+      print('Couldnt create funding offers: '+ str(e))
+  if response != None:
+      print ("NEW Offer: ", response.notify_info)
+      id = vars(response.notify_info)['id']
+      return(id)
+  else:
+      return(None)
+
+async def cancel_all_offers():
+    remove = []
+    for offer in billetera.bot.offers:
```

Comment:
billetera or wallet? why using Spanish and English mixed?

Reply:
Es codigo viejo eso, nose porque esta ahi. Era en español para no confundirlo con el objeto Wallet de bfx